src/modules/aws_vpc/entity/route_table.ts

```diff
@@ -0,0 +1,59 @@
+import { Column, Entity, JoinColumn, ManyToOne, OneToMany, PrimaryGeneratedColumn, Unique } from 'typeorm';
+
+import { cloudId } from '../../../services/cloud-id';
+import { AwsRegions } from '../../aws_account/entity';
+import { Route } from './route';
+import { RouteTableAssociation } from './route_table_association';
+import { Vpc } from './vpc';
+
+@Entity()
+export class RouteTable {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ nullable: true })
+  @cloudId
+  routeTableId?: string;
+
+  @ManyToOne(() => Vpc, {
+    nullable: false,
+    eager: true,
+    onDelete: 'CASCADE',
```

Comment:
Are we sure we want this cascade here? Also, does this mean deleting the RouteTable would also delete the VPC and Region records, or deleting one of those two records would delete the RouteTable?

Reply:
I believe this means deletion of VPC cascades to its RouteTables and it's expected.